app_demo.rb

```diff
@@ -0,0 +1,59 @@
+require './lib/order'
+class Takeaway
+
+  attr_accessor :menu1, :order1, :name, :price, :menu_item1
+
+  def initialize
+    @menu1 = Menu.new Dish
+  end
+
+
+  def demo
+    begin
+      puts 'Takeaway Challenge'
+      puts '=================='
+      puts
+      puts 'Select Option:'
+      puts 'A - Add dish to menu'
+      puts 'B - Display menu so that can make an order'
+      puts 'C - Create an order'
+      puts 'D - Total an order'
+      puts 'X - End'
+      option = gets.chomp
+      case option.downcase
+      when 'a'
+        while true
+          puts 'Add dishes name and then price (press return twice to end order )'
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)